resources/html/usermap.html

```diff
@@ -21,146 +21,140 @@
   </head>
   <body>
     <h3>Here is your checklist!</h3>
-    <!--The div element for the map -->
     <div id="map"></div>
     <script>
-      //the cookies are a global variable. they need to be parsed 
-      var decodedCookie = decodeURIComponent(document.cookie);
-      var cookiedata = decodedCookie.split(';');
+      //the cookies are a global variable. they need to be parsed into a JSON object
+      var decodedGlobalCookie = decodeURIComponent(document.cookie);
```

Comment:
Research whether this could be done somewhere else (backend, utils front-end situation, etc).